__main__.py

```diff
@@ -12,16 +12,25 @@ def main():
     before = time.time()
 
     args = sys.argv
-    SIM_DURATION_SECONDS = float(args[1])
-    scenario = args[2]
-    load = args[3]
+    NUMB_OF_SERVER = int(args[1])
+    SIM_DURATION_SECONDS = float(args[2])
+    scenario = args[3]
+    load = args[4]
 
-    if len(args) == 5:
-        results_path = args[4]
+    if len(args) >= 6:
```

Comment:
This is a problem, no?

You're checking if the value length is 6 or greater and is accessing the index 5.

Reply:
I think no. 
The sys.argv catchs the python file too, so we are considering it in our counting.
For example, if we type on command line "python3 __main__.py 1 100 control high results", the sys.argv become a list like this: "['__main__.py', '1', '100', 'control', 'high', 'results']"
I have decided to check if is equal or bigger than 6, because the next parameters could be opcional. 
I may be wrong, but I can't see the problem right now.